antismash/detection/hmm_detection/test/test_hmm_detection.py

```diff
@@ -158,7 +158,7 @@ def test_find_clusters(self):
 
     def test_create_rules(self):
         rules = hmm_detection.create_rules(self.rules_file, self.signature_names)
-        assert len(rules) == 52
+        assert len(rules) == open(self.rules_file).read().count("\nRULE")
```

Comment:
Doesn't this somewhat defeat the "check we didn't accidentally delete a rule" character of the test?

Reply:
I agree this test was quite cumbersome, so maybe we don't really want to use it that way.